modules/log-collector-rsyslog-server.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/cluster-logging-collector.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log-collector-rsyslog-server_{context}"]
+= Configuring the collector to listen for connections as a syslog server
+
+You can configure your log collector to collect journal format infrastructure logs from outside the cluster by specifying `syslog` as a receiver input in the `ClusterLogForwarder` custom resource (CR). This enables you to use a single common log store for journal format infrastructure logs that are collected from both inside and outside of your {product-title} cluster.
```

Comment:
We only accept logs inside the cluster.(To Be Accurate, as the limitation of the router, the syslog client must be same LAN).

Reply:
While it is true that the LB Service will not be created by the CLO, so by just running the CLF configuration, the Syslog receiver will not be available automatically to the outside world. However, it could be exposed by a LB Service, which is actually what OpenStack is doing to send logs from outside the cluster to the Syslog Receiver.

In this case, I would prefer to keep the text as it is now to reflect this capability.

Reply:
@jlarriba in that case, should we detail information here about creating the LoadBalancer Service as part of this procedure, and/or be more specific that this is for an OpenStack integration only if that's the only intended use case?

Reply:
I dont think we should detail that kind of information here. It is just Kubernetes knowledge.

Regarding being more specific about OpenStack, I don't really know if we need to be too specific here. I think what we already have (journal format logs that end up in infrastructure) is enough. No need to mention OpenStack in OCP docs.

Reply:
WFM! Thanks

Reply:
/lgtm